asapdiscovery-docking/asapdiscovery/docking/tests/test_rmsd_calculation.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+import numpy as np
+from pytest import skip
+
+skip("No OELicense in CI yet", allow_module_level=True)
+
+
+def test_rmsd_calculation():
+    """
+    This function contains two unit tests that check the correctness of the calculate_rmsd_openeye function.
+
+    The first test checks whether the RMSD between a reference molecule and itself is zero.
+    It does this by loading the reference molecule from an SDF file using the load_openeye_sdf function from the
+    asapdiscovery.data.openeye module and passing it twice to the calculate_rmsd_openeye function.
+    It then uses the assert statement to check that the calculated RMSD is equal to zero.
+
+    The second test checks whether the RMSD between a reference molecule and a query molecule is correct.
+    It does this by loading the reference molecule and the query molecule from SDF files using the load_openeye_sdf
+    function and passing them to the calculate_rmsd_openeye function.
+    It then uses the assert statement to check that the calculated RMSD is equal to the pre-determined value.
+    """
+    from asapdiscovery.data.openeye import load_openeye_sdf
+    from asapdiscovery.docking.analysis import calculate_rmsd_openeye
+
+    input_dir = Path("inputs")
+
+    ref_mol = load_openeye_sdf(str(input_dir / "Mpro-P0008_0A_ERI-UCB-ce40166b-17.sdf"))
+    query_mol = load_openeye_sdf(
+        str(input_dir / "ERI-UCB-ce40166b-17_Mpro-P2201_0A.sdf")
+    )
+
+    rmsd = calculate_rmsd_openeye(ref_mol, ref_mol)
+    assert rmsd == 0.0
+
+    rmsd = calculate_rmsd_openeye(ref_mol, query_mol)
+    assert rmsd == 5.791467472680422
```

Comment:
Not a blocker. Commonly comparing floats like this is prone to errors due to floating point precision. If the tests run for now is okay but we probably want in the future to use `np.isclose` or something similar to perform the comparison.